docs/source/development/pyrealm_build_data.md

```diff
@@ -16,22 +16,27 @@ kernelspec:
 
 The `pyrealm` repository includes both the `pyrealm` package and the
 `pyrealm_build_data` package. The `pyrealm_build_data` package contains datasets that
-are used in the `pyrealm` build process. This includes:
+are used in the `pyrealm` build and testing process. This includes:
 
 * Example datasets that are used in the package documentation, such as simple spatial
   datasets for showing the use of the P Model.
-* "Golden" datasets including a set of input data and output predictions from
-  previous implementations of `pyrealm` functionality. These are used to benchmark
-  `pyrealm` functionality within the testing framework (see below).
+* "Golden" datasets for regression testing `pyrealm` implementations against the outputs
+  of other implementations. These datasets will include a set of input data and then
+  output predictions from other implementations.
+* Datasets for providing profiling of `pyrealm` code and for benchmarking new versions
+  of the package code against earlier implementations to check for performance issues
 
 Note that `pyrealm_build_data` is a source distribution only (`sdist`) component of
-`pyrealm`, so is not included in binary distributions (`wheel`), such as those available
-from PyPi.
+`pyrealm`, so is not included in binary distributions (`wheel`) that are typically
+installed by end users. This means that the main `pyrealm` codebase should not require
+data from the `pyrealm_build_data` package.
```

Comment:
Is there anything automated in place to prevent this from happening?

Reply:
No - I've made it a bit more explicit. We probably could make some kind of a checker, but it seems a low probability event at the moment.